src/helpers/query/administrations.js

```diff
@@ -125,9 +128,25 @@ export const administrationPageFetcher = async (isSuperAdmin, exhaustiveAdminOrg
     undefined,
   );
 
+  const creatorIds = administrationData.map((adm) => adm.data.createdBy).filter(Boolean);
+  const uniqueCreatorIds = [...new Set(creatorIds)];
+  const creatorDocs = uniqueCreatorIds.map((id) => `${documentPrefix}/users/${id}`);
+  const { data: creators } = await axiosInstance.post(':batchGet', { documents: creatorDocs });
+  const creatorsData = creators.reduce((acc, { found }) => {
+    if (found) {
+      const creatorId = _last(found.name.split('/'));
+      acc[creatorId] = {
+        id: creatorId,
+        ..._mapValues(found.fields, (value) => convertValues(value)),
+      };
+    }
+    return acc;
+  }, {});
+
```

Comment:
Can we make a ticket to denormalize or normalize whichever it is the administration document to have the creator's name as well? 

Aritra also brought this up, and we should add these fields so we're not adding extra network calls.